packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1170,6 +1196,26 @@ export class AudiusAPIClient {
     return adapted
   }
 
+  async getCollectionMetadata({
+    collectionId,
+    currentUserId
+  }: GetCollectionMetadataArgs) {
+    this._assertInitialized()
+    const encodedCurrentUserId = encodeHashId(currentUserId)
+    const encodedCollectionId = this._encodeOrThrow(collectionId)
```

Comment:
is there a reason to use encode vs encodeOrThrow? Odd to have both here

Reply:
yeah no idea, just copied from the method below for playlists. ill just do encode honestly